recipes/libcnpy/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "libcnpy" %}
+{% set version = "0.0.20210729" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rogersce/cnpy/archive/4e8810b1a8637695171ed346ce68f6984e585ef4.tar.gz
+  sha256: 5120abc54a564efa92c642cc0199cc4fd3f345901157de9fbbdcedbb34d28d8a
+
+build:
+  number: 0
+  skip: true  # [win]
```

This needs a quite strict `run_exports` as packages build with this in host should also depend on the exact same version in `run`.
```suggestion
  skip: true  # [win]
  run_exports:
    - {{ pin_subpackage("libcnpy", max_pin="x.x.x") }}
```